setup.py

```diff
@@ -115,6 +118,13 @@ def setup_dependencies():
 
 if sys.argv[1] in ('install', 'develop', 'bdist_wheel'):
     setup_dependencies()
+    from torch.utils.cpp_extension import BuildExtension
```

Comment:
lazy  install cpp ext?

Reply:
When building source distribution, cpp extenstion won't be built, and only source code is packaged. Thus, torch is not required and we can use github's cluster to build, instead of self-hosted machine.